codex_of_the_damned/templates/archetypes/experimental/creeping-sabotage.html

```diff
@@ -0,0 +1,107 @@
+{% extends "archetypes/_layout.html" %}
+{% import "archetypes/experimental/creeping-sabotage.json" as deck_json %}
+
+{% block meta %}
+<meta property="decklist" content='{{ deck_json | tojson }}' />
+{% endblock %}
+
+{% block archetype_name %}
+<h1><img src="{{ url_for('static', filename='img/icons/clan-nosferatu.svg') }}"> Creeping Sabotage</h1>
+
+<div class="warning">
+    {% trans trimmed
+        creeping_sabotage=card("Creeping Sabotage"),
+		github=external("https://github.com/lionel-panhaleux/codex-of-the-damned", "Github")
+    %}
+	This is an experimental build starring {{ creeping_sabotage }}.
+	The decklist proposed here is subject to change before reaching its highest level of
+	performance. Feel free to contact us on our {{ github }} to help us better the concept.
+    {% endtrans %}
+</div>
+{% endblock %}
+
+{% block archetype %}
+{% trans trimmed
+	beast_leatherface=card("Beast, The Leatherface of Detroit"),
+	ebanezer_roush=card("Ebanezer Roush"),
+	gustave_brunnelle=card("Gustaphe Brunnelle"),
+
+	aid_from_bats=card("Aid from Bats"),
+	bums_rush=card("Bum's Rush"),
+	carrion_crows=card("Carrion Crows"),
+	creeping_sabotage=card("Creeping Sabotage"),
+	deep_song=card("Deep Song"),
+	earthshock=card("Earthshock"),
+	improvised_tactics=card("Improvised Tactics"),
+	slam=card("Slam"),
+	the_warrens=card("The Warrens"),
+	thrown_sewer_lid=card("Thrown Sewer Lid"),
+
+	actions=link("/strategy/fundamentals", _("Actions"), _anchor="AB"),
+	combat=link("/strategy/combat", _("Combat")),
+	intercept=link("/strategy/fundamentals", _("Intercept"), _anchor="stealth-and-intercept"),
+	modules=link("/strategy/deck-building", _("Modules"), _anchor="modules-and-density"),
+	range_management=link("/strategy/combat", _("Range management"), _anchor="range-management"),
+	stealth=link("/strategy/fundamentals", _("Stealth"), _anchor="stealth-and-intercept"),
+
+	nananimalism=link("/archetypes/nananimalism", "Nananimalism"),
+	euro_brujah=link("/archetypes/euro-brujah", "Euro Brujah"),
+
+	img_clan_nosferatu='<img src="%(url)s" />'|format(url=url_for('static', filename='img/icons/clan-nosferatu.svg'))|safe
+%}
+<h2>Highlights</h2>
+<p>
+	This is a <em>rush</em> deck based on mid-cap {{ POT }} vampires.
+</p>
+<p>
+	Ordinary rush deck uses {{ ANI }} as the key discipline because {{ deep_song }}
```

Comment:
Many rush decks are using POT (Enkidu, Brujah, Guruhi, Tupdogs). Some are using VAL or CEL. ANI rush is not so common, it's really just one build. A successful one, for sure.

Reply:
I think I'd just start talking about Creeping Sabotage right here in the beginning. It's the whole point of the decklist.
IMO the point is to use neither bleed nor vote to deliver pool damage - hence disabling a lot of defensive modules out there. It's slower, it costs a bit of blood, and the cards need to be defended. To compensate, we go mid-cap for speed and A&B.

You can then maybe talk a bit about how you got inspiration from ween ani but didn't want to go down that road because deep song means bleeding, which defeats the purpose of using sabotage to not bleed.